frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx

```diff
@@ -0,0 +1,804 @@
+
+import { ApplyDialog, LearnDialog, RequirementDialog } from "../Dialog";
+import CTAButtons from "../CallToAction/CTAButtons";
+
+const Home: React.FunctionComponent = () => {
+  const [isApplyDialogOpen, setApplyDialogOpen] =
+    React.useState<boolean>(false);
+  const [isLearnDialogOpen, setLearnDialogOpen] =
+    React.useState<boolean>(false);
+  const [isRequirementDialogOpen, setRequirementDialogOpen] =
+    React.useState<boolean>(false);
+  const intl = useIntl();
+  const quote = useQuote();
+
+  return (
+    <>
+      <div data-h2-overflow="b(x, hidden)">
+        <div
+          className="hero"
+          data-h2-position="b(relative)"
+          data-h2-overflow="b(all, hidden)"
+        >
+          <div
+            className="hero__content"
+            data-h2-position="b(absolute)"
+            data-h2-width="b(100)"
+          >
+            <div data-h2-container="b(center, full)">
+              <div
+                data-h2-padding="b(left-right, s) s(left-right, l)"
+                data-h2-display="b(flex)"
+                data-h2-flex-direction="b(column)"
+                data-h2-align-items="b(center)"
+                data-h2-justify-content="b(space-between)"
+              >
+                <div
+                  className="hero__logo"
+                  data-h2-margin="l(top, m)"
+                  data-h2-width="m(50)"
+                >
+                  <h1>
+                    <img
+                      data-h2-margin="b(top, m)"
+                      data-h2-width="b(100)"
+                      src={imageUrl(
+                        INDIGENOUSAPPRENTICESHIP_APP_DIR,
+                        `logo-${intl.locale}.svg`,
+                      )}
+                      alt={intl.formatMessage({
+                        defaultMessage:
+                          "IT Apprenticeship Program for Indigenous Peoples",
+                        description:
+                          "Homepage title for Indigenous Apprenticeship Program",
+                      })}
+                    />
+                  </h1>
+                </div>
+                <div
+                  className="hero__apply"
+                  data-h2-display="b(flex)"
+                  data-h2-justify-content="b(center)"
+                  data-h2-width="m(25)"
+                >
+                  <Button
+                    color="ia-primary"
+                    mode="solid"
+                    onClick={() => setApplyDialogOpen(true)}
+                    block
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Apply Now",
+                      description: "Button text to apply for program",
+                    })}
+                  </Button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <img
+            className="hero__image"
+            src={imageUrl(INDIGENOUSAPPRENTICESHIP_APP_DIR, "hero.jpg")}
+            alt=""
```

Comment:
A lot of alt tags in the is file are blank.  Are you just waiting for copy from the designers?

Reply:
All images outside of the logo were considered [decorative images](https://www.w3.org/WAI/tutorials/images/decorative/) so the `alt` attribute is left blank to not interrupt the flow of the document for assistive technology users.